### DISCUSSION TITLE
Фильтр по особенности трека (features)

### DISCUSSION THREAD

**Discussion by Bedmit** [Chimildic/goofy#181] [Помощь с алгоритмом] [answered]
На сколько я понимаю [Особенности трека (features)](https://chimildic.github.io/goofy/#/guide?id=%d0%9e%d1%81%d0%be%d0%b1%d0%b5%d0%bd%d0%bd%d0%be%d1%81%d1%82%d0%b8-%d1%82%d1%80%d0%b5%d0%ba%d0%b0-features) являются параметрами для генерации рекомендованных треков. А возможно ли их использовать в качестве фильтров в уже готовом массиве? 
Меня последнее время стали доставать всякие интро из болтовни, попадающие из альбомов в мой плейлист дня. С удовольствием бы их прибил с помощью _speechiness_

**Comment by Chimildic** [comment, marked as answer]
[Filter.rangeTracks](https://chimildic.github.io/goofy/#/reference/filter?id=rangetracks)

**Reply by Bedmit** [reply]
Действительно. И как сам не заметил?
